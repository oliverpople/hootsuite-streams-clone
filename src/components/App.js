import React, { Component } from 'react';

class App extends Component {
  constructor() {
      super();

      this.state = { streams: [] };
  }


  render() {
    return (
      <div>
        <h2>Hootsuite Streams</h2>
      </div>
    )
  }
}

export default App;
