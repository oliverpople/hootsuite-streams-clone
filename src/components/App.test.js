import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

const app = shallow(<App />);

it('renders correctly', () => {
  expect(app).toMatchSnapshot();
});

it('initialises the `state` with an empty list of streams', () => {
  expect(app.state().streams).toEqual([]);
})
