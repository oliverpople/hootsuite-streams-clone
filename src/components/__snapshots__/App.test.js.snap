// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h2>
        Hootsuite Streams
</h2>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Hootsuite Streams",
      },
      "ref": null,
      "rendered": "Hootsuite Streams",
      "type": "h2",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h2>
          Hootsuite Streams
</h2>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Hootsuite Streams",
        },
        "ref": null,
        "rendered": "Hootsuite Streams",
        "type": "h2",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
